lib/Translation/TranslationEntriesDumper/TranslationEntriesDumper.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Pimcore\Translation\TranslationEntriesDumper;
```

Comment:
```suggestion
namespace Pimcore\Translation;
```

There's no need to put that into a dedicated namespace, or what was the idea behind this? 😊 If there's no specific reason, I'd move it one level up. Thanks!

Reply:
a level above is only Translator and other services have separate namespaces, even Xliff12Escaper in Escaper - though it doesn't implement or extend something. So I did the same. But moved it one level up, done.